Mathlib/Algebra/GroupWithZero/Action/Defs.lean

```diff
@@ -327,7 +327,17 @@ def DistribSMul.toAddMonoidHom (x : M) : A →+ A :=
 
 end DistribSMul
 
-/-- Typeclass for multiplicative actions on additive structures. This generalizes group modules. -/
+/-- Typeclass for multiplicative actions on additive structures.
+
+For example, if `G` is a group (with group law written as multiplication) and `A` is an
+abelian group (with group law written as addition), then to give `A` a `G`-module
+structure (for example, to use the theory of group cohomology) is to say `[DistribMulAction G A]`.
+Note in that we do not use the `Module` typeclass for `G`-modules, as this is for modules
+over a ring rather than a group.
```

Comment:
```suggestion
Note in that we do not use the `Module` typeclass for `G`-modules, as the `Module` typclass
is for modules over a ring rather than a group.
```